fontforge/macbinary.c

```diff
@@ -1423,7 +1422,6 @@ static int DumpMacBinaryHeader(FILE *res,struct macbinaryheader *mb) {
 
 	/* Creation time, (seconds from 1/1/1904) */
     now = mactime();
-    time(&now);
```

Comment:
What's up with this?

Reply:
Oops, I forgot to add the background story in the commit. :)

That `time(&now)` has been added by GW during a mass commit. If the comment is right then using `time` there is plainly wrong, as `time` counts from 1970. On the other hand, `mactime` counts from 1904, just like the comment says. In addition, in the following line the modification time is also stored and there only `mactime` is used.

I trust the comment and the overall context, thus removed that `time`. However I have put it in a separate comment, so that it will be easy to bisect should any problem arise.